float32 z_axis

---

bool success